lib/adapters/index.js

```diff
@@ -3,8 +3,11 @@ var Juttle = require('../runtime/index').Juttle;
 var stochastic = require('./stochastic-adapter');
 var file = require('./file-adapter');
 var http = require('./http');
+var io = require('./io');
 
 // register the built-in adapters
 Juttle.adapters.register('file', file({}));
 Juttle.adapters.register('http', http({}));
 Juttle.adapters.register('stochastic', stochastic({}));
+Juttle.adapters.register('stdin', io.StdinAdapter({}));
```

Comment:
I was actually envisioning putting the two in one adapter (`io` or `terminal`) that implements both `read` and `write`, so the juttle would look like

```
read io  | write io;
// or
read terminal | write terminal
```

Thoughts?